AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_ExposureControlAdded.py

```diff
@@ -114,29 +216,17 @@ def AtomEditorComponents_ExposureControl_AddedToEntity():
             Tests.exposure_control_component,
             exposure_control_entity.has_component(AtomComponentProperties.exposure_control()))
 
-        # 3. UNDO the entity creation and component addition.
-        # -> UNDO component addition.
-        general.undo()
-        # -> UNDO naming entity.
-        general.undo()
-        # -> UNDO selecting entity.
-        general.undo()
-        # -> UNDO entity creation.
-        general.undo()
+        # 3. Remove the Exposure Control component.
+        exposure_control_component.remove()
         general.idle_wait_frames(1)
-        Report.result(Tests.creation_undo, not exposure_control_entity.exists())
+        Report.critical_result(Tests.exposure_control_component_removal,
```

Comment:
is this a critical_result? should the test terminate at this point if the component is not removed, because the next step is undo the removal and continue?